docs/application/vstools-mac/overview.md

```diff
@@ -1,150 +1,137 @@
-
-4. Enter the root directory path where you want to install and click **Next**.
-
-   ![Set the installation path](media/howtoinstall-installwizard3.png)
-
-   The Tizen SDK installer is downloaded and the baseline SDK is installed automatically.
-
-   ![Installer download](media/howtoinstall-installwizard4.png)
-
-   ![Baseline SDK installation](media/howtoinstall-installwizard5.png)
-
-   Finally, the Tizen Package Manager installs the Tizen SDK tools.
-
-   ![Tool installation](media/howtoinstall-installwizard6.png)
-
-<a name="set-tizen-baseline-sdk"></a>
-### Set the Tizen Baseline SDK Path 
-
-You can also use the Tizen Package Manager to set up the Tizen baseline SDK path or each tool path directly:
-
-- To set up the Tizen Baseline SDK path:
-  1. In the Visual Studio IDE menu, go to **Tools &gt; Tizen &gt; Tizen Package Manager**.
-  2. Select **Use installed Tizen SDK**.
-
-     ![Baseline SDK Install](media/howtoinstall-installwizard7.png)
-
-  3. Enter the root directory of your existing Tizen Studio installation.
-
-     ![Baseline SDK Install](media/howtoinstall-installwizard8.png)
-
-  > **Note**  
-  > If the installer gives a warning about your Tizen Studio version being too low, update Tizen Studio by using the Tizen Package Manager after setting the tool path.
-
-- To set up each tool path directly:
-  1. In the Visual Studio Mac IDE menu, go to **Project &gt; Solution Options &gt; Tizen &gt; Tools**.
-  2. Enter the root directory of your existing Tizen Studio installation in the **Tool Path** field.
-
-     ![Check the SDK tool path](media/howtoinstall-checktoolpath.png)
-
-     The other tools paths are automatically set up.
-
-### Install Emulator Images
-
-If you do not have a real device, you can run applications in the Tizen Emulator.
-
-To download emulator images, you can use the Tizen Package Manager or the Tizen Emulator Manager:
-
-- To use the Tizen Package Manager:
-  1. In the Visual Studio Mac IDE menu, go to **Tools &gt; Tizen &gt; Tizen Package Manager**.
-  2. Select the profiles and versions you want to install and click **Install**.
-
-     ![Package Manager](media/howtoinstall-packagemanager.png)
-
-- To use the Tizen Emulator Manager:
-
-  > **Note**
-  >
-  > The Tizen Emulator Manager shows the emulator images installation window only when no images are installed on your computer.
-
-  1. In the Visual Studio Mac IDE menu, go to **Tools &gt; Tizen &gt; Tizen Emulator Manager**.
-  2. Select the profiles and versions you want to install and click **OK**.
-
-     ![Emulator Manager](media/howtoinstall-emulatormanager.png)
-
-
-## Troubleshoot
-
-If you encounter problems with the installation:
-
-- To make sure the baseline Tizen SDK is installed correctly, check the tool path by selecting **Project &gt; Solution Options &gt; Tizen &gt; Tools**.
-
-  ![Check the SDK tool path](media/howtoinstall-checktoolpath.png)
-
-
+# Visual Studio for Mac Extension for Tizen

```

Comment:
I think the page title has to be changed.

```suggestion
# Install Visual Studio for Mac Extension for Tizen.
```

Please change the title for LNB.
File: https://github.com/Samsung/tizen-docs/blob/master/docs/application/toc_application.md

before |  after
--- | ---
... | ...
\# Visual Studio for Mac Extension for Tizen  | \# Visual Studio for Mac Extension for Tizen 
\#\# [Overview](/application/vstools-mac/overview.md) | \#\# [Install](/application/vstools-mac/overview.md)

Reply:
Done. Thank you.

Reply:
I'm sorry that my comment is old and there are a few mistakes.
```suggsion
# Install Visual Studio for Mac Extension for Tizen
```
I removed the period at the end of the page title.

File: https://github.com/Samsung/tizen-docs/blob/master/docs/application/toc_all.md

I updated toc_all.md  long time ago. 
We don't create a tree when the topic has no child. The menu tree looks as follow:
```
 # [Visual Studio for Mac Extension for Tizen](/application/vstools-mac/overview.md)
```